prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-periodics.yaml

```diff
@@ -1343,6 +1343,101 @@ periodics:
       - 'E2E_ARGS=--stress -run=TestStress*'
       - 'GCP_CLUSTER=stress-test'
 
+# Experimental prowjob to verify the new create-clusters flow on standard.
+# If stable, this is intended to replace the above testgroup-based definitions
+- name: standard-regular
+  interval: 1h
```

Comment:
do we want to run this every hour? how long does it usually take with 8 parallel? Should we start less frequent and ramp it up when we know?

Reply:
This is effectively the same parallelism into the existing tests which are sharded into 8 testgroups, so it theoretically should take approximately the average runtime of the current jobs. I haven't done extensive timing but the runtime is around an hour. The 1h interval in practice means the tests are always running